userpanel/modules/finances/style/bclean/templates/userbalancebox.html

```diff
@@ -13,6 +13,9 @@
 	.table-click[data-href] td:not(.buttons):hover {
 		cursor: pointer;
 	}
+	.table-thead {
+		color: white;
```

Comment:
bez sensu - przewala użycie koloru w inne miejsce - to nic nie daje.
Jeszcze raz napiszę (więcej już tego nie będę pisał): najlepiej, żeby zbierać znaczeniowo klasy css i umieszczać w style.css tematu graficznego `bclean` (jest takowy plik).

Reply:
Poprawione.

Reply:
Poprawione.